packages/api/src/api/contributions.ts

```diff
@@ -0,0 +1,58 @@
+import { Router } from 'express';
+import { Contribution } from '../entities/Contribution';
+import logger from '../logger';
+
+export const contributions = Router();
+
+contributions.get('', async (req, res) => {
+  if (req.user) {
+    try {
+      const allContributionsList = await req.entityManager.find(Contribution, {});
```

Comment:
In order to get just the contributions for the user we queried by, pass it in as a `where` argument:
```suggestion
      const allContributions = await req.entityManager.find(Contribution, { userId });
```